src/main/java/MainClass.java

```diff
@@ -1,18 +1,25 @@
-import data.ScriptingProject;
 import gui.RootPane;
 import javafx.application.Application;
-import xml.XmlWriter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class MainClass {
+    /**
+     * Add a logger to the application to keep track of all the things happening in the
+     * code. This logger has different levels and is able to distinguish between error
+     * levels.
+     */
+    static final Logger logger = LogManager.getLogger();
 
     /**
      *  @param args - the main arguments.
      */
     public static void main(String[] args) {
-        System.out.println("Welcome to the main class of goto-fail;");
+        logger.error("-------------------------------------------------------------------");
```

Comment:
The error is added to make the program start visible in the error loglevel, too. Otherwise, programs starts and restarts would not be visible in the error log level.